app/mutations/BaseMutation.ts

```diff
@@ -75,9 +75,9 @@ export default class BaseMutation<T extends BaseMutationType = never> {
             reject(error);
             if (failureMessage) {
               toast(failureMessage, {
-                className: 'mutation-toast Toastify__toast--error',
+                className: 'toastalert-error',
```

Comment:
We didn't need all these classes, and the `Toastify__toast` etc. ones are added automatically by react-toastify.